shared/js/ui/templates/grade-scorecard.es6.js

```diff
@@ -0,0 +1,65 @@
+const bel = require('bel')
+const siteRating = require('./shared/site-rating.es6.js')
+const siteRatingSubtitle = require('./shared/site-rating-subtitle.es6.js')
+
+module.exports = function () {
+  return bel`<section class="sliding-subview sliding-subview--has-fixed-header">
+   <div class="site-info site-info--full-height card">
+     <div class="hero border--bottom">
+      <a href="#" class="hero__close js-sliding-subview-close">
+        <span class="icon icon__arrow icon__arrow--large icon__arrow--left">
+        </span>
+      </a>
+      ${siteRating(
+        this.model.site.isCalculatingSiteRating,
+        this.model.site.siteRating,
+        this.model.site.isWhitelisted
+      )}
+      <h1 class="hero__title">${this.model.site.domain}</h1>
+      ${siteRatingSubtitle(
+        this.model.site.isCalculatingSiteRating,
+        this.model.site.siteRating,
+        this.model.site.isWhitelisted
+      )}
+    </div>
+    ${getReasons(this.model.reasons)}
+    ${getGrades(this.model.site.siteRating)}
+  </div>
+  </section>`
+}
+
+function getReasons (reasons) {
+  if (!reasons.length) { return }
```

Comment:
You don't need curly braces here.

Reply:
Also, please check `!reasons && !reasons.length`

Reply:
I've got into the habit of using curly braces even for tiny guards. Less worry if the guard ever becomes multiline or if I need to add errors/logs/debuger lines inside the guard.

Not too strongly attached to having them. I'll remove them here and the other place you pointed out.